### DISCUSSION TITLE
I've lost my e-mail with the key

### DISCUSSION THREAD

**Discussion by naynmarques** [aclap-dev/video-downloadhelper#1013] [Help and Q&A]
### Your question

Hi!

I lost my e-mail with the key. Can I receive again, please?

My e-mail is nayaranmarques@gmail.com

Thank you so much!

### Addon & CoApp version and details

Hi!

I lost my e-mail with the key. Can I receive again, please?

My e-mail is nayaranmarques@gmail.com

Thank you so much!

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

At the top of every page of this forum, find this link:

![#001](https://github.com/user-attachments/assets/1523c385-fb29-4b81-8ca5-296a96d4d758)

When you click that link, you will get a page that serves as the closest thing we have to a user manual.  You should get in the habit of referring to that regularly.  Part way down the page you will find this:

![#002](https://github.com/user-attachments/assets/769ece0d-8a0d-4b5b-bc14-ca7dce8d151a)

Your answer is in there.

**Reply by naynmarques** [reply]
Thankyou!
